federator/src/lib/Federator.js

```diff
@@ -6,6 +6,7 @@ const BridgeFactory = require('../contracts/BridgeFactory');
 const FederationFactory = require('../contracts/FederationFactory');
 const AllowTokensFactory = require('../contracts/AllowTokensFactory');
 const utils = require('./utils');
+const TOKEN_TYPE_COIN = 0;
```

Comment:
same comment as before - I'd maybe even make it a global constant somewhere appropriate (I mean the enum)